modules/setting-up-cpu-manager.adoc

```diff
@@ -172,8 +179,28 @@ QoS Class:       Guaranteed
 Node-Selectors:  cpumanager=true
 ----
 
-. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the `pause` process:
+. Verify that a CPU has been exclusively assigned to the pod by running the following command:
++
+[source,terminal]
+----
+# oc describe node --selector='cpumanager=true' | grep -i cpumanager- -B2
```

Comment:
What does this do? aren't you trying to find the Pod?

Reply:
Yes, we try to verify that a whole CPU is assigned to the cpu manager pod. It can be done with the above command.

Reply:
you can use oc exec commands:

`oc exec -it pods/<pod-name> -- bash -c "cat /sys/fs/cgroup/cpuset.cpus" `

which should give you the cpus assigned

Reply:
`# oc exec -it cpumanager-mlrrz -- bash -c "cat /sys/fs/cgroup/cpuset.cpus"
ERRO[0000] exec failed: unable to start container process: exec: "bash": executable file not found in $PATH
# oc exec -it cpumanager-mlrrz -- "cat /sys/fs/cgroup/cpuset.cpus"
ERRO[0000] exec failed: unable to start container process: exec: "cat /sys/fs/cgroup/cpuset.cpus": stat cat /sys/fs/cgroup/cpuset.cpus: no such file or directory
command terminated with exit code 255`

Reply:
try this:
```
[root@helix35 ~]# oc exec -it pods/pod1 cat /sys/fs/cgroup/cpuset.cpus
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
4,6
```

Reply:
The cpumanager-pod has only pause process without any filesystem. Therefore, there is no entry for sysfs. The above command also returns an error. Could you please try step.8 of the CPU manager documentation?
We also don't see the entry "/sys/fs/cgroup/cpuset/" in IBM Z architecture. Is it available on x86?

Reply:
I this approach system-Z specific, because otherwise I'd concur with @mrniranjan, checking the `cpuset.cpus` it simpler and safer (= more explicit guarantee about the correct allocation)